backend/src/main/java/de/sambam/gamenighttracker/security/JwtUtils.java

```diff
@@ -0,0 +1,34 @@
+package de.sambam.gamenighttracker.security;
+
+import de.sambam.gamenighttracker.service.TimeUtils;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.stereotype.Service;
+
+import java.sql.Date;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.HashMap;
+
+@Service
+public class JwtUtils {
+
+    private final TimeUtils timeUtils;
+    private final JwtConfig config;
+
+    public JwtUtils(TimeUtils timeUtils, JwtConfig config) {
+        this.timeUtils = timeUtils;
+        this.config = config;
+    }
+
+    public String createToken(String subject, HashMap<String, Object> claims) {
+        Instant now = timeUtils.now();
+        return Jwts.builder()
+                .addClaims(claims)
+                .setSubject(subject)
+                .setIssuedAt(Date.from(now))
+                .setExpiration(Date.from(now.plus(Duration.ofHours(4))))
```

Comment:
Think of a long long gamenight: 4h could be annoying. Maybe increase this a few hours?